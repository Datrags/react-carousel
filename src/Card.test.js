import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";

let testCard = <Card 
    caption="testing Caption" 
    src="https://i.imgur.com/h1pzaFN.jpeg" 
    currNum={1}
    totalNum={3}
/>
it("should render without crashing", () => {
    render(testCard);
    
});

it("should compare with snapshot", () =>{
    const {asFragment} = render(testCard);
    expect(asFragment()).toMatchSnapshot();
});