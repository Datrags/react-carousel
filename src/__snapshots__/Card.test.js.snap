// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should compare with snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      testing Caption
    </h4>
    <img
      alt="testing Caption"
      class="Card-image"
      src="https://i.imgur.com/h1pzaFN.jpeg"
    />
    <small
      class="Card-small"
    >
      Image 1 of 3.
    </small>
  </div>
</DocumentFragment>
`;
